dashboard/of-cloud-dashboard/handler.js

```diff
@@ -3,6 +3,28 @@
 const fs = require('fs');
 const request = require('request');
 
+const handleLogout = (context) => {
+  const headers = {
+    'Set-Cookie': [
+      'openfaas_cloud_token=',
+      `Expires=${new Date().toUTCString()}`,
```

Comment:
Just to make sure perhaps we should make this 1 minute or 10 seconds in the past? What if there is a time sync issue between client/server?